### PR TITLE
Add partition in morphtree

### PR DESCRIPTION
**Pull request by lidakanari**

### CODE REVIEW

File: neurom/analysis/morphtree.py [hunk 1]
```diff
@@ -300,6 +300,16 @@ def trunk_azimuth(tree, soma):
     return np.arctan2(vector[COLS.Z], vector[COLS.X])
 
 
+def partition(tree):
+    '''Ratio of maximum number of sections
```
Comment (juanchopanza) [inline neurom/analysis/morphtree.py]: Could you explain better what this returns?

**Comment by juanchopanza** [comment]
Which lines are you referring to? Could you add an inline comment in the right place?

File: neurom/analysis/morphtree.py [hunk 2]
```diff
@@ -300,6 +300,16 @@ def trunk_azimuth(tree, soma):
...
+       over minimum number of sections at
+       each bifurcation point.
+    '''
+    return [np.max([n_sections(i.children[0]), n_sections(i.children[1])]) /
+            np.min([n_sections(i.children[0]), n_sections(i.children[1])])
```
Comment (lidakanari) [inline neurom/analysis/morphtree.py]: Is there a better way to minimize the computation and repetition in the previous two lines?

Comment (lidakanari) [inline neurom/analysis/morphtree.py]: Lines 308-309 since it doesn't seem to put it in the right place.

Comment (juanchopanza) [inline neurom/analysis/morphtree.py]: You can define a helper function inside the function, to call n_sections only once per child:

    def partition_at_point(bif_point):
        n = n_sections(bif_point.children[0])
        m = n_sections(bif_point.children[1])
        return max(n, m) / min(n,m)

Also, no idea why you are using `np.max`. All it seems to do is force you to build lists for no reason.

Comment (lidakanari) [inline neurom/analysis/morphtree.py]: Ok, I was thinking to add this into morphmath if it makes sense and then call it from morphtree.

Comment (juanchopanza) [inline neurom/analysis/morphtree.py]: I think it relies on the tree structure, so it should probably stay in `morphtree`.

**Pull request merged**